docs/core/tracer.md

```diff
@@ -388,6 +388,10 @@ This is useful when you need a feature available in X-Ray that is not available
     --8<-- "examples/snippets/tracer/escapeHatch.ts"
     ```
 
+We only expose [a subset of the methods available](https://docs.powertools.aws.dev/lambda/typescript/latest/api/classes/_aws_lambda_powertools_tracer.provider_ProviderService.ProviderService.html) in the AWS X-Ray SDK for Node.js based on the most common use cases relevant for AWS Lambda and their compatibility with the Tracer utility.
```

Comment:
I would remove this paragraph.

1 - You are already explaining it in the first paragraph.
2 - "based on the most common use cases" I think it is vague and doesn't add much.

Reply:
Removed!